src/index.html

```diff
@@ -2,12 +2,59 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport"content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__block">
+      <div class="stars stars--0">
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--1">
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--2">
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--3">
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+      </div>
+
+      <div class="stars stars--4">
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
+        <span  class="stars__star"></span>
```

Comment:
Should be only 1 spaces
```suggestion
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
        <span class="stars__star"></span>
```